examples/visionos/HelloWorld/BUILD

```diff
@@ -1,22 +1,30 @@
-load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 load("@bazel_skylib//rules:build_test.bzl", "build_test")
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library",
+)
 load("//apple:visionos.bzl", "visionos_application")
 load("//apple:versioning.bzl", "apple_bundle_version")
 
+licenses(["notice"])
+
 swift_library(
     name = "Sources",
     srcs = [
-        "Sources/App.swift",
```

Comment:
I chose to adopt the upstream `HelloWorld` example, they're basically the same but decide to do this to ease future cherry-picks. Let me know what you think.